### ISSUE TITLE
inplace of browse library can we show some dropdown of categories .. on select on each dropdown we can add shapes into our library

### ISSUE THREAD

**Issue by Sohailamjad1** [excalidraw/excalidraw#4372]
here is the image [
![Screenshot (101)](https://user-images.githubusercontent.com/86312321/144806097-ed2be950-5c73-45e2-af65-c15fed12ba40.png)
](url)

**Comment by ad1992** [comment]
You mean prevent navigating away from current tab (excalidraw.com) to (libraries.excalidraw.com) ? We will be soon revamping the UI so you will be able to navigate to the libraries right inside the editor.

**Comment by Sohailamjad1** [comment]
Hi..
Thanks for the info.so we cant revamp from our side ?
________________________________

**Comment by Sohailamjad1** [comment]
> You mean prevent navigating away from current tab (excalidraw.com) to (libraries.excalidraw.com) ? We will be soon revamping the UI so you will be able to navigate to the libraries right inside the editor.

yes..i want to prevent navigating away  from current tab (excalidraw.com) to (libraries.excalidraw.com)

**Comment by ad1992** [comment]
> Hi.. Thanks for the info.so we cant revamp from our side ?

If you want to revamp you will have to fork the repo and update as per your need. You might have to fork the [excalidraw-libraries](https://github.com/excalidraw/excalidraw-libraries) as well and make changes.
But I would suggest to wait for the changes unless it's a blocker so that maintenance is easier for you as this would be significant change.

**Comment by Sohailamjad1** [comment]
sure..if  excalidraw  library navigate to same canvas then it would be better
